hw/bitstream/BUILD

```diff
@@ -161,3 +166,38 @@ bitstream_splice(
     for (otp_name, img_target) in get_otp_images()
     for authenticity in KEY_AUTHENTICITY
 ]
+
+# Create manifest fragments for all the cached bitstreams
+bitstream_fragment_from_manifest(
+    name = "chip_earlgrey_cw310_cached_fragment",
+    srcs = [
+        "@bitstreams//:chip_earlgrey_cw310_bitstream",
+        "@bitstreams//:chip_earlgrey_cw310_otp_mmi",
+        "@bitstreams//:chip_earlgrey_cw310_rom_mmi",
+    ],
+    design = "chip_earlgrey_cw310",
+    manifest = "@bitstreams//:manifest",
+    tags = ["manual"],
+)
+
+bitstream_fragment_from_manifest(
+    name = "chip_earlgrey_cw310_hyperdebug_cached_fragment",
+    srcs = [
+        "@bitstreams//:chip_earlgrey_cw310_hyperdebug_bitstream",
+        "@bitstreams//:chip_earlgrey_cw310_hyperdebug_otp_mmi",
+        "@bitstreams//:chip_earlgrey_cw310_hyperdebug_rom_mmi",
+    ],
+    design = "chip_earlgrey_cw310_hyperdebug",
+    manifest = "@bitstreams//:manifest",
+    tags = ["manual"],
+)
+
+bitstreams_cache_entry(
+    name = "cache_from_cache",
+    testonly = True,
+    srcs = [
+        ":chip_earlgrey_cw310_cached_fragment",
+        ":chip_earlgrey_cw310_hyperdebug_cached_fragment",
+    ],
+    tags = ["manual"],
+)
```

Comment:
As we discussed offline, this rule is never actually used correct?

The only problem I have with idle rules in our BUILD files is that they tend to break without anyone noticing until later.

Reply:
Aye, so we could remove it. At some point, we'd probably like to have better dep tracking, remote build execution, and build_settings to select cached or built, and _then_ this rule would be great for creating cache entries from within bazel. Unfortunately, we don't have any of that yet...

Reply:
Removed.